qa/acceptance/spec/lib/cli/logstash-plugin/install_spec.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
+
+shared_examples "logstash install" do |logstash|
+  before(:each) do
+    logstash.install(LOGSTASH_VERSION)
+  end
+
+  after :each do
+    logstash.uninstall
+  end
+
+  describe "on #{name}" do
+    context "with a direct internet connection" do
+      context "when the plugin exist" do
+        context "from a local `.GEM` file" do
+          let(:gem_path) { download_gem("logstash-filter-dns", "2.1.3") }
+          let(:gem_path_on_vagrant) { "/vagrant/vendor/download/logstash-filter-dns-2.1.3.gem" }
+          # after(:each) { FileUtils.rm_rf(gem_path) }
+
+          it "successfully install the plugin" do
+            command = logstash.run_command_in_path("bin/logstash-plugin install #{gem_path_on_vagrant}")
+            expect(command).to run_successfully_and_output(/Installation successful/)
```

Comment:
I wonder if this is something we should check inside the matcher? so in all places we check for successful install we have them grouped in a single matcher. This would be a new specific matcher for me, this one works also good. What do you think?